src/main/resources/application.conf

```diff
@@ -180,6 +180,8 @@ mantis {
     tx-pool-size = 1000
 
     pending-tx-manager-query-timeout = 5.seconds
+
+    transaction-timeout = 2.minutes
```

Comment:
Super minor: Why 2 minutes?

Reply:
@AlanVerbner  I think it's a reasonable default, but I'm open to other suggestions

Reply:
@LukasGasior1 actually your value seems to be quite good. Maybe we can think of this as a number of blocks function like RSK guys and leave a comment explaining the rationale like: https://github.com/rsksmart/rskj/blob/c739053e23ef26baa43f069afb20d32fff1710b7/rskj-core/src/main/resources/config/rsk-sample.conf#L89

wdyt?

Reply:
120/14 gives around 9 blocks, pretty similar

Reply:
yeah, I was suggesting to leave a comment behind the rationale of that value so it doesn't look arbitrary for someone that might look at it